src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/locations/LocationsToNomisIntTest.kt

```diff
@@ -102,7 +102,7 @@ class LocationsToNomisIntTest : SqsIntegrationTestBase() {
       fun `will callback back to location service to get more details`() {
         waitForCreateProcessingToBeComplete()
 
-        locationsApi.verify(getRequestedFor(urlEqualTo("/locations/$DPS_ID")))
+        locationsApi.verify(getRequestedFor(urlEqualTo("/locations/$DPS_ID?includeHistory=false")))
```

Comment:
The sync process ignores location change history but the reconciliation checks it